requirements/base.txt

```diff
@@ -2,7 +2,7 @@
 
 numpy>=1.16.4
 torch>=1.3
-tensorboard>=2.2.0
+tensorboard==2.2.0
```

Comment:
Why is it 2.2.0 here and 2.2.1 in the other file?

Reply:
Conda didn't have tensorboard==2.2.0. We could change the requirements/base.txt to also be 2.2.1, but I figured I'd keep it to what it was.

Reply:
you are using ==, are you sure? It is basically saying none of the earlier versions work with PL.

Reply:
Thats what >=2.2.0 states anyways, this just prevents newer versions as well

Reply:
ok I see, sorry but the linked issue does not state why this must be. I had to ask because it is hard for me to believe only this particular version works. If there is a particular version that nees to be blocked, one could do 
`>=2.2.0, !=2.x.y `
or 
`>=2.2.0, <2.x.y`

Reply:
It was decided freeze TB until 0.9 release, as some versions above 2.2.0 (not sure which ones, definitely 2.3.0) have issues with the tensorboard logger

Reply:
we shall use =2.2 for conda and ~=2.2 for pip